.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ concurrency:
 
 env:
   CARGO_TERM_COLOR: always
-  RUST_VERSION: 1.73.0
+  RUST_VERSION: 1.74.0
```

Comment:
Idk if we need https://github.com/FuelLabs/fuel-core/pull/1656 if we want to just bump these here.